lib/form1010_ezr/service.rb

```diff
@@ -153,10 +150,20 @@ def post_fill_veteran_ssn(parsed_form)
       parsed_form
     end
 
+    def post_fill_veteran_gender(parsed_form)
+      return if parsed_form['gender'].present?
+
+      StatsD.increment("#{Form1010Ezr::Service::STATSD_KEY_PREFIX}.missing_gender")
+
+      parsed_form['gender'] = @user.gender
+      parsed_form
+    end
+
```

Comment:
🎨 Not to echo CodeClimate here, but with this many postfilled fields now, would it make sense to abstract this pattern to a more general `post_fill_field(parsed_form, field_name)` or something?

If we're expecting this code to all go away soon™ it might not be worth it, but this debugging has been going on for a fair bit already....

Reply:
Yes, this makes sense! I cleaned up the code, so please feel free to re-review!